region/crateria/east.json

```diff
@@ -98,29 +124,55 @@
                   "notable": false,
                   "requires": ["SpaceJump"]
                 },
-                {
-                  "name": "Pass Below",
-                  "notable": false,
-                  "requires": ["h_canBombThings"],
-                  "note": "In reality, bombs lead to 1 and not 3, but 1->3 and 3->1 are free so it doesn't matter."
-                },
                 {
                   "name": "Gravity",
                   "notable": false,
                   "requires": [
                     "Gravity",
-                    "canWalljump"
+                    {"or": [
+                      "canWalljump",
+                      "canGravityJump",
+                      {"and": [
+                        "canSpringBallJumpMidAir",
+                        "HiJump"
+                      ]}
+                    ]}
                   ]
                 },
                 {
                   "name": "Moat Reverse Jump",
                   "notable": false,
-                  "requires": [ "canCarefulJump" ],
+                  "requires": [
+                    "canDisableEquipment",
+                    "canCarefulJump",
+                    {"or": [
+                      {"doorUnlockedAtNode": 2},
+                      "canTrickyJump",
+                      "canLateralMidAirMorph",
+                      "SpeedBooster",
+                      {"adjacentRunway": {
+                        "fromNode": 2,
+                        "usedTiles": 2
+                      }}
+                    ]}
+                  ],
                   "note": [
                     "Execution of this strat is non-trivial, and failing will lead to falling into the pit.",
                     "Depending on item loadout, that could be a softlock."
                   ],
                   "devNote": "Could use a failure definition?"
+                },
+                {
+                  "name": "Shinespark",
+                  "notable": false,
+                  "requires": [
+                    {"canComeInCharged": {
+                      "framesRemaining": 0,
+                      "fromNode": 2,
+                      "shinesparkFrames": 42,
+                      "excessShinesparkFrames": 15
+                    }}
+                  ]
```

Comment:
```suggestion
                  ],
                  "devNote": "FIXME: This will grab the item and continue the spark to the left door. If this is an E-Tank, Samus will not maintain full Energy after the strat."
```